packages/client/src/config.schema.json

```diff
@@ -531,6 +461,47 @@
                     "minItems": 1
                 }
             }
+        },
+        "peerDescriptor": {
+            "type": "object",
+            "additionalProperties": false,
+            "required": [
+                "id"
+            ],
+            "properties": {
+                "id": {
+                    "type": "string"
+                },
+                "type": {
+                    "$ref": "#/definitions/nodeType"
+                },
+                "websocket": {
+                    "$ref": "#/definitions/connectivityMethod"
+                }
+            }
```

Comment:
Should these fields of `JsonPeerDescriptor` be in the schema:
```
openInternet?: boolean
region?: number
```

Reply:
At least for now, these fields are not useful in client side configuration.

Reply:
Would it make sense to remove these fields from the client-side configuration type (`NetworkPeerDescriptor`)?